action.yml

```diff
@@ -29,7 +29,7 @@ runs:
     - name: Action
       shell: powershell
       run: |
-        $secure_password = ConvertTo-SecureString -String "${{ inputs.service-account-password }}" -AsPlainText -Force
+        $secure_password = ConvertTo-SecureString -String '${{ inputs.service-account-password }}' -AsPlainText -Force
```

Comment:
Single quotes in the password will still break this.

Reply:
See https://github.com/im-open/register-windows-service/pull/5 for an alternative.